phoenix-core/src/main/java/org/apache/phoenix/execute/MutationState.java

```diff
@@ -894,6 +897,7 @@ private void send(Iterator<TableRef> tableRefIterator) throws SQLException {
 
         MultiRowMutationState multiRowMutationState;
         Map<TableInfo, List<Mutation>> physicalTableMutationMap = Maps.newLinkedHashMap();
+        Map<TableInfo, TableRef> physicalLogicalMap = Maps.newLinkedHashMap();
```

Comment:
Nit: Please rename this to logicalToPhysicalMap .. this one is easy to be read as 'physicalToLogicalMap' and being passed to filterIndexCheckerMutations() where logicalToPhysicalMap is expected.

Reply:
Actually, it's physical to Logical, I renamed it accordingly. :)